### ISSUE TITLE
Query regarding the cgan paper

### ISSUE THREAD

**Issue by ajaykmr2292** [junyanz/pytorch-CycleGAN-and-pix2pix#438]
In the paper "**Image to Image Translation Using Conditional Adversarial Networks**", under the section **Data requirements and speed**,  its given that 

> the day to night training set consists of only 91 unique webcams (see results in Figure 14).

I would like to know How was cgan trained on such a small number of data points? Was it like the data points were increased to 20k (or more) using augmentation and then the model was trained?

**Comment by junyanz** [comment]
I think day2night is slightly different from your colorization case. There are 91 webcam videos, but each video has many frames. For each webcam, we created all day photos ("night" attribute < 0.1) to all night photos. ("night" attribute > 0.8).Unfortunately, you still only have ~90 different natural scenes no matter what you do. Therefore, a larger dataset might be better. 

I think in general more images should be more helpful. People usually train colorization models on millions of images. See [colorization-pytorch](https://github.com/richzhang/colorization-pytorch) for more details.

**Comment by ajaykmr2292** [comment]
So, for day2night, was ~90 different natural scenes (which maybe like 1000s of images obtained from every frame) sufficient for the model to learn? I have this query because even in the colorization case, I just have 170 images (which are then augmented to create 10k images). But then model is not learning in colorization.

If day2night is able to learn from just 90 different scenes and colorization is not able to learn from 170 images (10k after augmentation), then is it like the learning that colorization model has to do is much more than that for day2night? (I mean is it like the colorization network has to learn about various features like face and their color as opposed to day2night case where that much learning is not required?)

And by more data, do you mean different dataset (not the ones obtained from augmenting)? Is there anything I could do to train the model using these small set of images?

Thanks.

**Comment by junyanz** [comment]
I will say that 90 is not enough for day2night. Sometimes the model doesn't work for new types of natural scenes, which are not present in the training set. For your case, you may want to see if the model works during training vs. during the test. The model should perform well in the training test at least before we are talking out test time performance. You should do augmentation (flip, cropping, jitter the color, [etc](https://github.com/mdbloice/Augmentor).). You might want to use dropout. (I assume that you have used it)
